content/applications/sales/sales/amazon_connector/manage.rst

```diff
@@ -1,131 +1,172 @@
-============================
-Manage Amazon orders in Odoo
-============================
+=======================
+Amazon order management
+=======================
 
-Synchronization of orders
-=========================
+Order synchronization
+=====================
+
+Orders are automatically fetched from Amazon, and synchronized in Odoo, at regular intervals.
 
-Orders are automatically fetched from Amazon and synchronized in Odoo at regular intervals.
 The synchronization is based on the Amazon status: only orders whose status has changed since the
-last synchronization are fetched from Amazon. For **FBA** (Fulfilled by Amazon), only **Shipped**
-and **Canceled** orders are fetched. For **FBM** (Fulfilled by Merchant), the same is done for
-**Unshipped** and **Canceled** orders. For each synchronized order, a sales order and a customer are
-created in Odoo if they are not yet registered.
+last synchronization are fetched from Amazon.
+
+For *FBA* (Fulfilled by Amazon), only *Shipped* and *Canceled* orders are fetched.
 
-When an order is canceled in Amazon and was already synchronized in Odoo, the corresponding sales
-order is automatically canceled in Odoo.
+For *FBM* (Fulfilled by Merchant), the same is done for *Unshipped* and *Canceled* orders. For each
+synchronized order, a sales order and customer are created in Odoo (if the customer is not already
+registered in the database).
 
 .. note::
-   To force the synchronization of an order whose status has not changed since the last
-   synchronization, activate the :ref:`developer mode <developer-mode>`, navigate to your Amazon
-   account and modify the date under :menuselection:`Orders Follow-up --> Last Order Sync`. Pick a
-   date anterior to the last status change of the order that you wish to synchronize and save.
+   When an order is canceled in Amazon, and was already synchronized in Odoo, the corresponding
+   sales order is automatically canceled in Odoo.
+
+Force synchronization
+=====================
+
+In order to force the synchronization of an order, whose status has **not** changed since the
```

Comment:
not changed...in Odoo or Amazon?
It's important to be specific here if we're going to evoke **bold** emphasis like that